crates/router/src/configs/settings.rs

```diff
@@ -127,6 +128,27 @@ pub struct DummyConnector {
     pub refund_retrieve_tolerance: u64,
 }
 
+#[derive(Debug, Deserialize, Clone)]
+pub struct Mandates {
+    pub supported_payment_methods: SupportedPaymentMethodsForMandate,
+}
+
+#[derive(Debug, Deserialize, Clone)]
+pub struct SupportedPaymentMethodsForMandate(
+    pub HashMap<enums::PaymentMethod, SupportedPaymentMethodTypesForMandate>,
```

Comment:
Do we need this hashmap be built through config or can there be a global static? I understand that having a config based approach will let us turn things on/off during startup, but the config structure keeps getting bloated, which gets cloned, passed to each actix thread.

Reply:
I agree that config structure is getting messy, but making it a global static will require re-deploying the application if any changes in the config right?